snuba/utils/threaded_function_delegator.py

```diff
@@ -26,40 +28,50 @@ def __init__(
         self,
         callables: Mapping[str, Callable[[], TResult]],
         selector_func: Callable[[TInput], Tuple[str, List[str]]],
-        callback_func: Optional[Callable[[List[Tuple[str, TResult]]], None]],
+        callback_func: Optional[Callable[[List[Tuple[str, TResult, float]]], None]],
```

Comment:
Seems about time to declare the result as a NamedTuple, so it  is easier to understand what each field means.

Reply:
I ended up changing it to a dataclass to keep mypy happy